cli/users.go

```diff
@@ -38,16 +38,30 @@ var cmdUsers = []cobra.Command{
 		},
 	},
 	{
-		Use:   "get <user_auth_token>",
+		Use:   "get [all | <user_id>] <user_auth_token>",
 		Short: "Get user",
 		Long:  `Returns user object`,
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) != 1 {
+			if len(args) != 2 {
 				logUsage(cmd.Use)
 				return
 			}
-
-			u, err := sdk.User(args[0])
+			pageMetadata := mfxsdk.PageMetadata{
+				Email:    "",
```

Comment:
Email is missing

Reply:
Yes. Because currently it is used to get all users. Our cli doesn't have the capability to get a specific user based on the email

Reply:
What is this "list all users with e-mail addresses"? Why do we provide e-mail addresses if we are listing all users?

Reply:
No, we aren't doing that. That is why the email field is empty for listing all users. There is a separate command to list a user by providing their email address.